src/cbapi/response/models.py

```diff
@@ -1407,6 +1407,30 @@ def group_by(self, field_name):
             log.debug("group_by only supported in Cb Response 6.1+")
             return self
 
+    def max_children(self, num_children):
+        """Sets the number of children to fetch with the process
+
+        This method is only available for Cb Response servers 6.0 and above. Calling this on a Query object connected
+        to a Cb Response 5.x server will simply result in a no-op.
+
+        :default: 15
+        :param int num_children: Number of children to fetch with process
```

Comment:
Does this service have a means of specifying "no limit on number of children"? Such as by setting max_children to 0 or a negative number? If so, it ought to be documented.

Reply:
Nope but that is a good point I should check for negative

Reply:
Never mind the API handles that for me it will just return zero results if negative or zero

Reply:
Pity it doesn't have an override to remove all limits. Oh well, I guess you can just pass in 99999 or some suitably high number if you want to have an effective "no limit."